### ISSUE TITLE
Duplication of Bottles

### ISSUE THREAD

**Issue by DarianLStephens** [Valiec/DeeperCaves#39]
####What type of issue is this? (e.g. bug, feature request)
Incompatibility/Other Issue

####Issue Description:
Crafting with Water Bottles places a new Empty Bottle in your inventory, regardless of if the bottle is consumed with the recipe.
####Steps to Reproduce: (if applicable)
Install this mod alongside any other mod which adds a recipe containing a water bottle which produces an output in a bottle; craft that recipe, and the bottle will be duplicated.
####System info: (only needed for bug reports)
Almost Certainly Irrelevant
Java version:
Irrelevant?
Minecraft version:
1.7.10
Deeper Caves version:
1.0.0
Other mods (if any):
Enviromine, for the recipes involving Water Bottles.
Crash report:
N/A.

**Comment by DarianLStephens** [comment]
Is there some way to disable that? I haven't found any reference to bottles in the code, so I don't know why this is happening.

**Comment by DarianLStephens** [comment]
Has the issue at least been found? This would be a lot of fun to explore with friends, but I still don't like having that problem.

**Comment by Valiec** [comment]
Sorry, I got busy with other things and almost forgot this. I'll investigate this as soon as I have time.

**Comment by DarianLStephens** [comment]
I appreciate that you'll look in to this. I'd love to at least know what's causing it.
It's not exclusively enviromine, it's any mod with a recipe using water bottles.
Vanilla just doesn't show it because it doesn't have a recipe like that (As far as I recall).

**Comment by Valiec** [comment]
Oops! I forgot this, I'm not sure at all why this is happening though, I'll likely add a simple recipe using a water bottle to test at some point.

**Comment by DarianLStephens** [comment]
Oh wow, it's been almost 10 years since this issue was opened!
Is this mod coming back, or something?

**Comment by Valiec** [comment]
Yeah, I've been working on a bunch of changes recently! It has a Discord server now, check the Modrinth page.

**Comment by DarianLStephens** [comment]
I always wanted to do a playthrough with this, and... maybe should have, but there were a few issues with it back in 1.7.
I remember some weird generation, and performance issues?
I can only imagine how interesting it would be in modern MC, though, especially with Immersive Portals so you can descend the full 2 kilometers without the game dying by loading a 2 km-tall chunk!
